DependencyInjection/Compiler/AddDependencyCallsCompilerPass.php

```diff
@@ -50,21 +51,31 @@ public function process(ContainerBuilder $container)
         foreach ($container->findTaggedServiceIds('sonata.admin') as $id => $tags) {
             foreach ($tags as $attributes) {
                 $definition = $container->getDefinition($id);
+                $parentDefinition = $definition instanceof DefinitionDecorator ? $container->getDefinition($definition->getParent()) : null;
```

Comment:
Not sure the new syntax is the right ?
php-cs-fixer doesn't see anything to fix ;)